src/Application/View/Extension/DocumentExtension.php

```diff
@@ -62,11 +62,35 @@ public function getFilters(): array
     public function getDocumentIcon(string $filename): string
     {
         /** @var array $CFG_GLPI */
-        global $CFG_GLPI;
+        /** @var \DBmysql $DB */
+        global $CFG_GLPI, $DB;
 
-        $icon = sprintf('/pics/icones/%s-dist.png', strtolower(pathinfo($filename, PATHINFO_EXTENSION)));
+        $extension = strtolower(pathinfo($filename, PATHINFO_EXTENSION));
 
-        return $CFG_GLPI['root_doc'] . (file_exists(GLPI_ROOT . $icon) ? $icon : '/pics/timeline/file.png');
+        if (!isset($CFG_GLPI['extension_icon'])) {
```

Comment:
It's unusual to have a $CFG_GLPI key set long after the init of GLPI.
If you don't plan to use it elsewhere, maybe just use a static variable.